cmd/otelcol/config/collector/agent_config.yaml

```diff
@@ -53,6 +53,11 @@ receivers:
       processes:
       # System processes metrics, disabled by default
       # process:
+      #   metrics:
+      #     process.cpu.utilization:
```

Comment:
Could you please add a comment indicating that these 2 metrics are not enabled by default by should be enable if one wants to use the process panel?